src/rest_api/classes/transcript/widgets/feature.clj

```diff
@@ -0,0 +1,23 @@
+(ns rest-api.classes.transcript.widgets.feature
+  (:require
+    [datomic.api :as d]
+    [rest-api.classes.feature.core :as feature]
+    [rest-api.classes.generic-fields :as generic]))
+
+(defn associated-features [transcript]
+  (let [db (d/entity-db transcript)
+        data  (->>
```

Comment:
think it's best to always have an expression at the start of a threaded macro, since to my eyes it looks like something is missing (!) e.g:

```clojure
(->> something
     something-else)
```

Reply:
good idea. I see what you are saying